src/net/atlas/download.rs

```diff
@@ -446,6 +446,7 @@ impl AttachmentsBatchStateContext {
                 let request = AttachmentRequest {
                     sources,
                     content_hash: content_hash.clone(),
+                    block_height: self.attachments_batch.block_height,
```

Comment:
Is `self.attachments_batch.block_height` is the current Stacks block height, is it?

Reply:
It looks like it. The value here seems to originate from the block_height field in `StacksHeaderInfo`